### ISSUE TITLE
GCDevExchange to Update Twitter Feed when new opportunities are published

### ISSUE THREAD

**Issue by samperd** [canada-ca/devex#38]
As a GCDevExchange User, Developer or Opportunity Promoter
I want GCDevExchange to update a dedicated twitter feed
So that I can subscribe to the twitter feed as a user and also promote my opportunity as a promoter stakeholder

Details:
* A dedicated twitter account would be useful
* Using an existing hash like #GCDevExchange (created by @ToferC )  https://twitter.com/hashtag/GCDevExchange?src=hash

**Comment by markwooff** [comment]
@sdkibb something you could build in .NET fairly easily?

**Comment by sdkibb** [comment]
Yes, it's already created. Just need the Twitter usernames, content of the tweets in both languages and we're set I believe.

**Comment by samperd** [comment]
well that seemed easy. What is still left to do?  Can I help with anything?

**Comment by samperd** [comment]
Great stuff and Good work.
